packages/vite/src/node/build.ts

```diff
@@ -246,15 +246,15 @@ export function resolveBuildOptions(raw?: BuildOptions): ResolvedBuildOptions {
     // Support browserslist
     // "defaults and supports es6-module and supports es6-module-dynamic-import",
     resolved.target = [
-      'es2019',
+      'es2020',
       'edge88',
       'firefox78',
       'chrome87',
       'safari13.1'
     ]
   } else if (resolved.target === 'esnext' && resolved.minify === 'terser') {
     // esnext + terser: limit to es2019 so it can be minified by terser
-    resolved.target = 'es2019'
+    resolved.target = 'es2020'
```

Comment:
On the other hand, should we be removing terser? And go all in to esbuild. Probably another discussion for team meeting.

Reply:
Could default to esbuild and let user optionally install parcel css or terser

Reply:
plugin-legacy uses terser, so I feel it is difficult to remove it.
I have created a PR which changes terser to optional dep. (#8049)